### PR TITLE
Auto generate change log entry

### LINKED ISSUE #99: Automatically add CHANGELOG.md to terra-dev-site
### Description of Issue
The generator automatically creates a terra-dev-site entry for the new component, but it doesn't generate an entry for the CHANGELOG.md.

#### Additional Details
We have decided to include CHANGELOGs alongside each component's doc page.
https://github.com/cerner/terra-core/issues/1616

### Steps to Reproduce the Issue
1. Generate a new component using `generator-terra-module` 
2. See that the CHANGELOG is not displayed on the terra-dev-site entry for that new component.

### Expected Outcomes
* Create the CHANGELOG entry on terra-dev-site for newly generated components.

### PR DESCRIPTION
**Pull request by tomleewu**
### Summary
Added feature to generate the Change Log onto the terra-dev-site entry of new components.

Closes #99.

### CODE REVIEW

File: generators/app/templates/src/terra-dev-site/ChangeLog.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/no-webpack-loader-syntax, import/first, import/extensions */
```
Comment (emilyrohrbough) [inline generators/app/templates/src/terra-dev-site/ChangeLog.jsx]: Do we need this eslint-disable line? What are we disabling?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ------------------
+### Added
+* Generate CHANGELOG entry in docs
```
Comment (emilyrohrbough) [inline CHANGELOG.md]: Can we update this to indicate we added a Changelog template for the terra-dev-site docs setup

File: test/app.js [hunk 3]
```diff
@@ -84,12 +85,12 @@ describe('generator-terra-module:app', function () {
...
 
         assert.fileContent(index, `import React from 'react';`);
-        assert.fileContent(index, `import DefaultMonsterCookiesSrc from '!raw-loader!../../../src/terra-dev-site/doc/example/DefaultMonsterCookies.jsx';`);
-        assert.fileContent(index, `import ReadMe from '../../../docs/README.md';`);
-        assert.fileContent(index, `import { version } from '../../../package.json';`);
+        assert.fileContent(index, `import DefaultMonsterCookiesSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/DefaultMonsterCookies.jsx';`);
+        assert.fileContent(index, `import ReadMe from '../../../../docs/README.md';`);
+        assert.fileContent(index, `import { version } from '../../../../package.json';`);
         assert.fileContent(index, `export default DocPage;`);
       });
```
Comment (emilyrohrbough) [inline test/app.js]: Can we add a test case to verify the import path of the Changelog? These paths are important for correct resolution and would need updated if structure changes in the future

Comment (emilyrohrbough) [reply test/app.js]: With this, can we also verify the `src` prop is receiving the correct expected git url?

Comment (tomleewu) [reply test/app.js]: Done in https://github.com/cerner/generator-terra-module/pull/100/commits/f01c02e101e519a71a66c81536b05738415dde27

Comment (emilyrohrbough) [reply test/app.js]: Thanks! I've ran into generator test failures by small tweaks before and was glad to have them caught in the tests!

**Pull request merged**